document/index.js

```diff
@@ -1,7 +1,36 @@
 const utils = require('./utils')
+const fetch = require('node-fetch')
+
+const defaultConfig = {
```

Comment:
ah, I see :) thx!

Reply:
Do i still need to answer the questions above about the defaults? :D
Spares some of the boilerplate and we "suggest" to use those ones in the readme anyway!